laghos.cpp

```diff
@@ -757,7 +782,7 @@ int main(int argc, char *argv[])
 
         samplerTimer.Start();
         if (usingWindows) {
-            outfile_twp.open("twpTemp.csv");
+            outfile_twp.open(outputPath + "/twpTemp.csv");
```

Comment:
Yes, twpTemp.csv will be outputted to the outputPath of the offline phase. Then online and restore phase will also need to use -k to find the file in the same outputPath. If -k isn't used, offline will default to writing to the run directory, and online and restore will default to reading from the run directory. So if -k isn't used, the only thing that's changed is instead of writing to the main directory, we write to the run directory.

Reply:
@kevinhkhuynh Thank you. Is this ready to be reviewed again?

Reply:
@chldkdtn Ready for review and passes all tests.